dateutil/test/test_rrule.py

```diff
@@ -4912,3 +4912,11 @@ def testWeekdayReprWithN(self):
 
         for repstr, wday in zip(with_n_reprs, with_n_wdays):
             self.assertEqual(repr(wday), repstr)
+
+    def testRaisesCorrectErrorForUnsupportedProperty(self):
+        rule = "X-ABC-MMSUBJ:19970902T090000"
+        self.assertRaisesRegex(ValueError, "unsupported property", rrulestr, rule, dtstart=datetime(1997, 9, 2, 9, 0))
```

Comment:
The CI test fails because `assertRaisesRegex` was only added in python 3. Since we still support python 2.7 we need to add an import and use that instead.

```python
from six import assertRaisesRegex
```

```suggestion
        assertRaisesRegex(ValueError, "unsupported property", rrulestr, rule, dtstart=datetime(1997, 9, 2, 9, 0))
```

Also, the test is in the class `WeekdayTest`. Move it to `RRuleTest` instead.